src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -61,7 +61,17 @@ private with sharing class ADDR_Addresses_TEST {
     /*********************************************************************************************************
     * @description The list of created test Addresses.
     */
-    private static list<Address__c> listAddrT;           
+    private static list<Address__c> listAddrT;
+    
+    private static ID orgRecTypeID;
+    private static ID householdRecTypeID;
+    private static ID adminRecTypeID;
+    
+    public static void recTypesSetup() {
```

Comment:
Yeah, I'm not going to hold this up for that, but I do think there's a better way to handle it. We're using a pattern from 2007 right now.  I don't think including the types is the right answer though - if we're storing the relevant type in a setting somewhere, we can retrieve it that - I can't recall, is that in there?